internal/services/containers/kubernetes_addons.go

```diff
@@ -25,6 +25,7 @@ const (
 	ingressApplicationGatewayKey    = "ingressApplicationGateway"
 	openServiceMeshKey              = "openServiceMesh"
 	azureKeyvaultSecretsProviderKey = "azureKeyvaultSecretsProvider"
+	webApplicationRoutingKey        = "ingress/webApplicationRouting"
```

Comment:
This is not working.
```
Error: creating Cluster: (Managed Cluster Name "example-aks1" / Resource Group "example-resources"): containerservice.ManagedClustersClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="AddonInvalid" Message="Addon ingress/webApplicationRouting is invalid." Target="AddonProfiles"
```

Understanding this comment should help to fix this issue:
https://github.com/Azure/azure-cli-extensions/blob/54fed385b1118eb1cb3fd8d95ba04d13ab4a9e38/src/aks-preview/azext_aks_preview/_consts.py#L137